bundles/org.openhab.binding.somfytahoma/src/main/java/org/openhab/binding/somfytahoma/internal/SomfyTahomaHandlerFactory.java

```diff
@@ -152,6 +153,15 @@ protected ThingHandler createHandler(Thing thing) {
         if (thingTypeUID.equals(THING_TYPE_GATE)) {
             return new SomfyTahomaGateHandler(thing);
         }
+        if (thingTypeUID.equals(THING_TYPE_ELECTRICITYSENSOR)) {
```

Comment:
Does it make sense to use an `if .. else if ...` structure here?

Reply:
changed to if .. else if ..